src/schema/v2/me/myBids.ts

```diff
@@ -379,3 +396,33 @@ function withSaleInfo(sale): SaleSortingInfo {
     liveBiddingStarted,
   }
 }
+
+function formatGravityLotStandingsToMatchCausalityLotStandings(
+  gravityLotStandings
+): LotStandingResponse {
+  return gravityLotStandings.map((lotStanding) => {
+    const { sale_artwork, id, bidder, highest_bid_amount_cents } = lotStanding
```

Comment:
since the lot standing is a 'synthetic' object, it has the following properties: https://github.com/artsy/gravity/blob/1ffe27dc7d5a55b36757ed1cf8998ca64b963f49/app/services/lot_standing_service.rb#L37-L40

(so no `id` or `highest_bid_amount_cents`). In general the `lotState` object prob needs a few more things added to it!